bot/cogs/sync/syncers.py

```diff
@@ -131,7 +131,7 @@ def _reaction_check(
             await message.edit(content=f':ok_hand: {mention}{self.name} sync will proceed.')
             return True
         else:
-            log.warning(f"The {self.name} syncer was aborted or timed out!")
+            log.trace(f"The {self.name} syncer was aborted or timed out!")
```

Comment:
Well I still think it should be visible in production logs so I disagree with it being anything lower than `info`. Someone looking at logs may not necessarily be looking at Discord to see the edited message.